submarine-server/server-api/src/main/java/org/apache/submarine/server/api/spec/ExperimentSpec.java

```diff
@@ -63,6 +64,14 @@ public CodeSpec getCode() {
   public void setCode(CodeSpec code) {
     this.code = code;
   }
+  
+  public Map<String, String> getExperimentHandlerSpec() {
+    return experimentHandlerSpec;
+  }
+
+  public void setExperimentHandlerSpec(Map<String, String> experimentHandlerSpec) {
+    this.experimentHandlerSpec = experimentHandlerSpec;
+  }
```

Comment:
I think we could add some more test cases to the `submarine-server/server-submitter/submitter-k8s/src/test/java/org/apache/submarine/server/submitter/k8s/ExperimentSpecParserTest.java` to pass the codecov checks.

Reply:
good call.